WordPress/Classes/ViewRelated/Jetpack/Install/JetpackRemoteInstallViewController.swift

```diff
@@ -109,9 +119,16 @@ extension JetpackRemoteInstallViewController: JetpackRemoteInstallStateViewDeleg
             return
         }
 
+        let installJetpack: JetpackInstallBlock = { [weak self] url, username, password, event in
```

Comment:
I think should be a proper function declaration (nested functions work in Swift, fwiw).

Reply:
To be honest, if this is only used in two places, we could just duplicate those two lines of code to avoid overcomplicating it :)

Reply:
I can simplify it duplicating the line with a different event name.
Do we have preferences over nested vs closures? I don't really like the nested declaration as style. I always preferred anonymous functions instead.

Reply:
I don't like either, really, and prefer a proper method.